File: Assets/Game/Hoftix/Script/GenerateScripts/UIModules/LoginPanelView.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class LoginPanelView:UIViewInterface
    {
        public UnityEngine.GameObject LoginPart=null;
		public ZJYFrameWork.UISerializable.LoginPartView LoginPartView=null;
		public ZJYFrameWork.UISerializable.RegisterPartView RegisterPartView=null;
		public ZJYFrameWork.UISerializable.LoginTapToStartView LoginTapToStartView=null;
		public ZJYFrameWork.UISerializable.LoginController LoginController=null;
		public UnityEngine.GameObject tips=null;
		public UnityEngine.UI.Text UserNameText=null;
		


        public void Init(UIView _view)
        {
            LoginPart=_view.GetObjType<UnityEngine.GameObject>("LoginPart");
			LoginPartView=_view.GetObjType<ZJYFrameWork.UISerializable.LoginPartView>("LoginPartView");
			RegisterPartView=_view.GetObjType<ZJYFrameWork.UISerializable.RegisterPartView>("RegisterPartView");
			LoginTapToStartView=_view.GetObjType<ZJYFrameWork.UISerializable.LoginTapToStartView>("LoginTapToStartView");
			LoginController=_view.GetObjType<ZJYFrameWork.UISerializable.LoginController>("LoginController");
			tips=_view.GetObjType<UnityEngine.GameObject>("tips");
			UserNameText=_view.GetObjType<UnityEngine.UI.Text>("UserNameText");
			
        }
    }
}

File: UnityMMOClient/Lua/application/app/ui/generateScripts/UIModules/LoginPanelView.lua
---@class LoginPanelView
LoginPanelView = class("LoginPanelView")
function LoginPanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.bg_Image = self._UIView:GetObjTypeStr("bg_Image") or UnityEngine.UI.Image
	self.Gonggao_Button = self._UIView:GetObjTypeStr("Gonggao_Button") or UnityEngine.UI.Button
	self.tips = self._UIView:GetObjTypeStr("tips") or UnityEngine.GameObject
	self.UserNameText = self._UIView:GetObjTypeStr("UserNameText") or UnityEngine.UI.Text
	self.LoginPart = self._UIView:GetObjTypeStr("LoginPart") or UnityEngine.GameObject
	self.LoginPart_UISerializableKeyObject = self._UIView:GetObjTypeStr("LoginPart_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.RegisterPart_UISerializableKeyObject = self._UIView:GetObjTypeStr("RegisterPart_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.LoginStart_CanvasGroup = self._UIView:GetObjTypeStr("LoginStart_CanvasGroup") or UnityEngine.CanvasGroup
	self.LoginStart_UISerializableKeyObject = self._UIView:GetObjTypeStr("LoginStart_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.AudioSource = self._UIView:GetObjTypeStr("AudioSource") or UnityEngine.AudioSource
end

